### DISCUSSION TITLE
What's the algorithm used for drawing lines?

### DISCUSSION THREAD

**Discussion by DMClVG** [nesbox/TIC-80#1988] [General]
I know it's not Bresenham and even looking through the code was futile. Does somebody know?

**Comment by nesbox** [comment]
Just usual linear drawing, it's not bresenham since we are using line to draw triangles and it doesn't work very accurately.

**Reply by DMClVG** [reply]
Alright, thanks!
Just one more thing given you're the maintainer: there's this fantasy console I've been [working on](https://github.com/DMClVG/TAC-70) that aims to not only be compatible with cartridges made with TIC-80 but to also extend it in many ways (networking API, higher resolution mode, extended palette, ..) 

For now it's for the most part a drop in replacement for running TIC-80 cartridges with most of the API still missing, but eventually I want this to be a easy to use Rust framework (in the likes of macroquad and ggez) but for lower resolution pixel perfect retro-esque games that allows you to write the game logic either in Rust or in a scripting language (Lua, Rhai, Fennel, ..)
It's still very WIP, but can run a few games originally made for the TIC-80.

Any thoughts on this ?
